spring-cloud-gcp-pubsub/src/main/java/com/google/cloud/spring/pubsub/core/PubSubProperties.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.spring.pubsub.core;
+
+/**
+ * Intermediate class to hold subscription-specific properties.
+ */
+class PubSubProperties {
```

Comment:
If it's only a wrapper for subscription properties, we should name it `PubSubSubscriptionProperties`. If we have to keep this intermediate class, there will be another one for the publisher properties, right?